src/StaticFAQ/FAQCategoryList.js

```diff
@@ -60,7 +60,17 @@ const querySubcategory = graphql`
 class FAQCategoryList extends React.Component<Props> {
   renderFAQArticlePerexes = (faqs: FAQArticle_article[]) => {
     return (
-      <div>{faqs.map(faq => <FAQArticle key={faq.id} article={faq} />)}</div>
+      <div>
+        {faqs.map(faq => (
+          <Link
+            key={faq.id}
+            to={`${routeDefinitions.STATIC_FAQ}/${faq.id}`}
```

Comment:
shouldn't this be `routeDefinitions.FAQ_ARTICLE` ?

Reply:
aah no, my bad, I confused `<FAQArticle/>`with `<FAQArticleDetail/>` sorry

Reply:
I think you can also wrap with `Link` the `<FAQArticle />` on `SearchAllFAQs`

Reply:
fixed